pkg/testers/ginkgo/ginkgo.go

```diff
@@ -76,6 +78,8 @@ func (t *Tester) Test() error {
 		"--kubectl-path=" + t.kubectlPath,
 		"--ginkgo.skip=" + t.SkipRegex,
 		"--ginkgo.focus=" + t.FocusRegex,
+		"--ginkgo.label-filter=" + t.LabelFilter,
```

Comment:
adding it with an empty value is a noop? I think I asked this several times but never remember it

Reply:
Its a noop, check the`pull-kubetest2-kind-build-up-down` job